src/main/java/api/ApiFeature.java

```diff
@@ -0,0 +1,27 @@
+package api;
+
+public enum ApiFeature {
```

Comment:
move this class out of the api package. the api package will hold the generic classes. so create a salesforce package and move this ApiFeature. Also I think you could named it as Endpoint, SalesforceEndpoint or Salesforce. something tha